CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ _None._
 
 ### Breaking Changes
 
-- Re-implement a few reader model types in Swift. [#556]
+- Re-implement a few reader model types in Swift. [#556, #557, #558]
```

Comment:
If I'm not mistaken, this would be an un-breaking change, so we could probably move this out of the `Breaking Changes` section (and thus avoid cutting a new major version unnecessarily?)

Reply:
Technically speaking, I'd argue they are still breaking changes, even if we revert the two properties changes [I mentioned above](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/562#discussion_r1049153503). The types are changed from Objective-C classes to Swift classes. And Swift class has its restrictions that Objective-C class don't have, like they cann't be inherited by an Objective-C class. I guess it's not likely that the apps have a subclass of these model types, but I'd say it's best to follow semantic version strictly, what do you think?